routes/list.js

```diff
@@ -0,0 +1,28 @@
+// returns a table listing all registered reports:
+exports.routes = {
+  method: 'GET',
+  path: '/',
+  config: {
+    plugins: {
+      'hapi-transform-table': {}
+    }
+  },
+  handler(request, h) {
+    const server = request.server;
+    let base = server.settings.app.routePrefix ?
+      `${request.info.host}/${server.settings.app.routePrefix}`
+      : `${request.info.host}`;
+    base = request.url.host === 'https' ? `https://${base}` : `http://${base}`;
```

Comment:
I don't think host will ever equal https, I think you want protocol.  Probably can just remove host and just use relative paths like `/reports/blah`